apps-rendering/src/components/Byline/AnalysisByline.tsx

```diff
@@ -0,0 +1,60 @@
+import { css } from '@emotion/react';
+import { background } from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	headline,
+	neutral,
+	remSpace,
+	until,
+} from '@guardian/source-foundations';
+import type { Option } from '@guardian/types';
+import { darkModeCss } from 'styles';
+import { DefaultByline } from './Byline.defaults';
+
+const blogStyles = css`
+	${headline.medium({
+		fontWeight: 'light',
+	})}
+
+	color: ${neutral[46]};
```

Comment:
Can we add this to `editorialPalette` as `text.byline`?

Reply:
Definitely, will move this up.